Makefile

```diff
@@ -130,23 +135,25 @@ fast-build-coverage: coverage
 
 # Build docker images
 .PHONY: build-images tag-images push-images push-images-to-kind install-nginx install-kubernetes-dependencies wait-kubernetes-dependencies
-	
+
 build-images: ## Build docker images
 	$(DOCKER-COMPOSE) -f ./docker-compose/docker-compose.yml build
 
 tag-images: ## Tag docker images with git hash and branch name
-	$(DOCKER) tag docker-compose_workflow-service:latest $(ORG)$(WORKFLOW_SERVICE_IMAGE):$(GIT_HASH)
-	$(DOCKER) tag docker-compose_notification-service:latest $(ORG)$(NOTIFICATION_SERVICE_IMAGE):$(GIT_HASH)
+	$(eval TAG?=$(GIT_HASH))
+	$(DOCKER) tag docker-compose_workflow-service:latest $(ORG)$(WORKFLOW_SERVICE_IMAGE):$(TAG)
+	$(DOCKER) tag docker-compose_notification-service:latest $(ORG)$(NOTIFICATION_SERVICE_IMAGE):$(TAG)
 
-	$(DOCKER) tag docker-compose_workflow-service:latest $(ORG)$(WORKFLOW_SERVICE_IMAGE):$(GIT_BRANCH)
-	$(DOCKER) tag docker-compose_notification-service:latest $(ORG)$(NOTIFICATION_SERVICE_IMAGE):$(GIT_BRANCH)
+	$(DOCKER) tag docker-compose_workflow-service:latest $(ORG)$(WORKFLOW_SERVICE_IMAGE):$(TAG)
```

Comment:
If this TAG we need to change in `.github/workflows/`

Reply:
The changes do not influence current behavior. Please see

> $(eval TAG?=$(GIT_HASH))

We still use hash and we only use TAG when we run `release-all` target